README.md

```diff
@@ -35,27 +35,30 @@ Current Status:
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test storage write buffer over page boarder-0 | 1      | 0      | OK     | 0.37               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test storage write byte above end address-0   | 1      | 0      | OK     | 0.25               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test storage write over the upper bound-0     | 1      | 0      | OK     | 0.18               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test storage write subsequent bytes-0         | 1      | 0      | OK     | 6.25               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test storage write without SD-0               | 2      | 0      | OK     | 0.35               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-advanced | Storage test true-0                                   | 1      | 0      | OK     | 0.05               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write 1.5 words-0                | 1      | 0      | OK     | 0.34               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write 2 words-0                  | 1      | 0      | OK     | 0.34               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write 3 byte-0                   | 1      | 0      | OK     | 0.32               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write buffer-0                   | 1      | 0      | OK     | 0.43               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write byte-0                     | 1      | 0      | OK     | 0.32               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write existing fails-0           | 1      | 0      | OK     | 0.44               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write half word-0                | 1      | 0      | OK     | 0.33               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write non-aligned-0              | 1      | 0      | OK     | 0.33               |
+| NRF52_DK-GCC_ARM | NRF52_DK      | tests-storage-nrf52-basic    | Storage test storage write word-0                     | 1      | 0      | OK     | 0.32               |
 +------------------+---------------+------------------------------+-------------------------------------------------------+--------+--------+--------+--------------------+
 ```
 
 ## TODO
 
 - add automated tests on dev kit hardware
-- store and erase haveto check if the softdevice is enabled, or not
```

Comment:
is this done?

Reply:
yes it is done. store and erase do not only check, if the softdevice is enabled, they also enable and disable the softdevice themselves, if it is not enabled.